src/applications/personalization/dashboard/components/health-care/Appointments.jsx

```diff
@@ -22,29 +26,18 @@ export const Appointments = ({ appointments, hasError }) => {
     locationName = nextAppointment?.providerName;
   }
 
-  if (hasError) {
-    return (
-      <div className="vads-u-display--flex vads-u-flex-direction--column large-screen:vads-u-flex--1 vads-u-margin-bottom--2p5">
-        <va-alert status="error" visible class="vads-u-margin-top--0">
-          <h3 slot="headline">We can’t access your appointment information</h3>
-          <p>
-            We’re sorry. Something went wrong on our end, and we can’t access
-            your appointment information. Please try again later or go to the
-            appointments tool.
-          </p>
-          <p>
-            <CTALink
-              text="Schedule and manage your appointments"
-              href="/health-care/schedule-view-va-appointments/appointments"
-            />
-          </p>
-        </va-alert>
-      </div>
-    );
-  }
```

Comment:
Is the error handling being done somewhere else, if so then we should be able to remove this prop from the component/prop types. If not, then we should probably still handle it here.

Reply:
Nevermind I see the HealthcareError component